pages/pipelines/rules.md

```diff
@@ -0,0 +1,169 @@
+# Pipeline rules
+
+Rules is a Buildkite feature used to manage and organize permissions between Buildkite resources.
+
+Rules express that an action is allowed between a source resource (e.g. a pipeline) and a target resource (e.g. another pipeline). Rules allow you to break out of the defaults provided by Buildkite such as cluster boundaries.
+
+## Pipeline rules best practices
+
+### How should I use rules?
+
+Use rules to manage permissions between pipelines and other resources. Rules are typically used in tandem with [clusters](/docs/clusters/overview) to increase security and control, where clusters set hard boundaries and rules provide exceptions.
+
+### Available rules
+
+#### `pipeline.trigger_build.pipeline` 
+
+Cross-cluster pipeline triggering (eg. allow a pipeline in one cluster to trigger a pipeline in another cluster).
+
+Value fields:
+
+- `triggering_pipeline_uuid` The UUID of the pipeline that is allowed to trigger another pipeline.
+- `triggered_pipeline_uuid` The UUID of the pipeline that is allowed to be triggered by the `triggering_pipeline_uuid` pipeline.
+
+### `pipeline.artifacts_read.pipeline` Allowing jobs in a pipeline to access artifacts from builds in other pipelines.
+
+Value fields:
+
+- `source_pipeline_uuid` The UUID of the pipeline that is allowed to access artifacts from another pipeline.
+- `target_pipeline_uuid` The UUID of the pipeline that is allowed to have its artifacts accessed by jobs in the `source_pipeline_uuid` pipeline.
+
+### Example rule use
```

Comment:
This example feels like it should go underneath the `pipeline.trigger_build.pipeline` section. 
Also, we dont have an example for `artifacts_read` - it might be nice to add one.